probe-rs/src/debug/variable.rs

```diff
@@ -518,11 +521,14 @@ impl Variable {
                         }
                     } else if self.type_name == VariableType::Struct("None".to_string()) {
                         "None".to_string()
+                    } else if matches!(self.type_name.clone(), VariableType::Array{entry_type: _,  count} if count == 0)
```

Comment:
```suggestion
                    } else if matches!(&self.type_name, VariableType::Array{count, ..} if count == 0)
```

This clone should not be necessary if done correct :) (most likely you can just reference the typename :)

Reply:
Done :)